src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2289,6 +2327,81 @@ components:
             example: 
             - "path/to/file1"
             - "path/to/file2"
+    JobQueue:
+      description: Structure of jobs queue
+      type: object
+      properties:
+        jq_pk:
+          description: Job queue id
+          type: string
+        jq_job_fk:
+          description: Job id
+          type: string
+        jq_type:
+          description: Job queue type
+          type: string
+        jq_args:
+          description: Any arguements
+          type: string
+        jq_starttime:
+          description: Job queue start time
+          type: string
+        jq_endtime:
+          description: Job queue end time
+          type: string
+        jq_endtext:
+          description: Job queue completion status
+          type: string
+        jq_itemsprocessed:
+          description: Number of items processed
+          type: string
+        jdep_jq_depends_fk:
+          description: Job queue dependency id
+          type: string
+        depends:
+          description: List of dependencies
+          type: array
+          items:
+            type: string
```

Comment:
Aren't both properties `jdep_jq_depends_fk` and `depends` give out same information in different format?

Reply:
Yes, Actually I wasn't sure about that as `depends` is an array, and I thought there can be multiple dependencies. Should I remove one of these?

Reply:
[UPDATED] Done!